packages/server/src/services/server.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Application } from 'express';
+import { createServer as createHttpServer } from 'http';
+import { createServer as createHttpsServer, ServerOptions } from 'https';
+import { readFileSync } from 'fs-extra';
+import path from 'path';
+
+export default class HttpsServerService {
+  private getAssets(): ServerOptions {
+    try {
+      return {
+        key: readFileSync(path.resolve('assets/server.key'), 'utf8'),
+        cert: readFileSync(path.resolve('assets/server.crt'), 'utf8'),
```

Comment:
Suggest we move these to the home directory of the user that's running the service. And use paths like path.resolve('~/server.key') here. In this case you can also use it locally (windows also has user home directories).

Reply:
Sounds OK for me

Reply:
Guess you should use 
path.resolve(process.env['HOME'] + '/server.key')
instead